terraform/modules/front_door/ip_list.tf

```diff
@@ -7,7 +7,12 @@ locals {
     "82.163.115.98/32",
     "87.224.105.250/32",
     "87.224.116.242/32",
-    "45.150.142.210/32"
-    # DLUHC - TODO
+    "45.150.142.210/32",
+    # DLUHC
+    "35.176.187.166/32"
+  ]
+  ip_allowlist_test = [
```

Comment:
I'm a bit suspicious of the need for this, but it is what DLUHC are telling us so lets go for it and can adjust it later if need be.